Cython/Utility/Coroutine.c

```diff
@@ -1881,7 +1906,8 @@ static PyType_Slot __pyx_CoroutineType_slots[] = {
     {Py_tp_methods, (void *)__pyx_Coroutine_methods},
     {Py_tp_members, (void *)__pyx_Coroutine_memberlist},
     {Py_tp_getset, (void *)__pyx_Coroutine_getsets},
-    {Py_tp_getattro, (void *) PyObject_GenericGetAttr},
+    {Py_tp_getattro, (void *)__pyx_Coroutine_getattro},
+    {Py_tp_setattro, (void *)__pyx_Coroutine_setattro},
```

Comment:
Could you run a quick micro-benchmark to see if this has any visible impact on the time it takes to look up and call `.send()` from Python code (e.g. from within `asyncio`)? Especially with all the attribute lookup optimisations in Py3.1[34]? Just so that we have a number.

Reply:
Will do (although it'll probably have to wait a few days).

There's also a Limited API 3.8 breakage that needs fixing for this PR

Reply:
Just calling `.send()` on a generator looks like it ends up about 3x slower. That seems fairly bad so I'm going to try to implement an alternative approach instead.